### PR TITLE
FEATURE: Add xslt to package feed

### PR DESCRIPTION
**Pull request by Sebobo**

### CODE REVIEW

**Comment by Sebobo** [comment]
We could also add some links or recommendations for feed readers.

File: DistributionPackages/Neos.MarketPlace/Resources/Public/Assets/feed.xsl [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                    }
+
+                    .site-footer {
+                        grid-area: footer;
+                        padding-top: 1rem;
+                        border-top: 1px solid #eee;
+                    }
+                </style>
+            </head>
+            <body style="font-family:'Work Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;">
```
Comment (jonnitto) [inline DistributionPackages/Neos.MarketPlace/Resources/Public/Assets/feed.xsl]: What do you think about
```suggestion
            <body style="font-family:-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"">
```
Some more infos: https://css-tricks.com/snippets/css/system-font-stack/

Comment (Sebobo) [reply DistributionPackages/Neos.MarketPlace/Resources/Public/Assets/feed.xsl]: Actually this css should also go into the top declaration. 

What are those fonts for?

Comment (Sebobo) [reply DistributionPackages/Neos.MarketPlace/Resources/Public/Assets/feed.xsl]: Ah sorry, now saw your link

**Comment by Sebobo** [comment]
I tried to put the CSS into a separate file and reference it according to the specs but the browser didn't load it. That's why I inlined it.

**Comment by Sebobo** [comment]
@jonnitto if you have actually design improvements they are very welcome. I first wanted it to work properly ;)

**Pull request merged**